ToDoList-version1/TodoList/src/main/java/com/example/demo/controller/ToDoListController.java

```diff
@@ -0,0 +1,62 @@
+package com.example.demo.controller;
+
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import com.example.demo.model.ToDoList;
+import com.example.demo.service.ToDoListService;
+
+@Controller
+public class ToDoListController {
+    @Autowired
+	private ToDoListService todolistService;
+   
+    @GetMapping("/todolist")
```

Comment:
Đọc hiểu thêm phần Restful api để dùng cho chính xác hơn nhé (cách đặt tên, method nào dùng trường hợp nào,...)
https://viblo.asia/p/thiet-ke-restful-api-nhu-the-nao-MdZGAYdyGox